aries-site/src/pages/components/pagination.mdx

```diff
@@ -1,90 +1,65 @@
-
-If there is only one page of items, it is optional to display the pagination component and summary of results.
-
-### Placement of pagination component
-
-The pagination component should be placed underneath and justified to the end of the content it is paginating.
-
-### Showing summary of results
-
-It may be beneficial to provide users with a summary of what range of results they are viewing. This should take the format of "Showing x-y of z" where x is the start of the page, y is the end of the page, and z is the total number of records. This should be left-aligned in a row with the pagination component.
-
-The results summary should be justified to the start of the row and the pagination component should remain justified to the end of the row.
-
-## Accessibility
+## Use cases
 
-Below are some accessibility requirements that ensure pagination is accessible for screen readers and keyboard navigation.
+### Managing datasets
 
-If your team is developing with Grommet, these considerations are already handled, so no further action is required from your team.
+Common ways to display datasets include, DataTable, List & Cards. Pagination can be attached to any of these options, allowing the user to browse the returned data in smaller segmented views.
 
-1. Wrap pagination links in a `<nav>` element.
-1. Provide an `aria-label="pagination navigaton"` and `role="navigation"`.
-1. Individual page buttons should have an `aria-label` that reads "Go to page X", where X is the page that will be navigated to upon selection.
-1. The active page button should have `aria-current="true"`.
-1. The next/previous arrow buttons should have `aria-label` that reads "Go to next page" and "Go to previous page", respectively.
-
-For more detailed documentation on these requirements, you can read more about [accessible pagination](https://www.a11ymatters.com/pattern/pagination/).
-
-## Variants
-
-In addition to [DataTable](/components/datatable#paginated) and [List](/templates/lists#paginated), here is another way pagination can be implemented.
+<Example
+  width="100%"
+  code={[
+    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/togglegroup/ToggleGroupViews.js',
+    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/togglegroup/CardView.js',
+    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/togglegroup/ListView.js',
+    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/togglegroup/utils.js',
+  ]}
+>
+  <ToggleGroupViews />
+</Example>
 
-### Paginated table
+## Anatomy
 
-Tables often contain large amounts of data. To improve the efficiency that a user can navigate to a specific item or range of items, use pagination.
+<Example
+  caption="Diagram illustrating the parts composing Pagination."
+  plain
+>
+  <PaginationAnatomy />
+</Example>
```

Comment:
In the deploy preview, the anatomy seems a bit off:
<img width="1025" alt="Screenshot 2024-07-25 at 12 12 34" src="https://github.com/user-attachments/assets/67428bde-a424-4be7-bbc4-4bf6f73739a2">

- If I am not mistaken @jcfilben had mentioned that the "line" above the container should not be there
- The numbers don't align with the part they are showing
- The StepOptions is centralized

Reply:
It was suggested that we leave the top border in the figma component as an affordance for product designers as they pagination was hard to define from the table. I'm fine with leaving it here for this example since its in the figma component.


Regarding the number/line alignment - It works fine on page load, but can be fragile when resining browser & using inspect element - this would be uncommon user behaviour. I don't think its a blocker to publication.

Reply:
@jcfilben @britt6612 since we want Pagination to be implemented with the top border, I'd lean towards leaving it in (and add it to the example at the top of the page) so devs can see how we recommend applying it.

Reply:
Should the hpe theme include a top border in `theme.pagination.container` so that people don't have to manually add it?

Reply:
Just looked at the code, I'm fine with how it is working currently